Documentation/contributor-guide/triage_issues.md

```diff
@@ -48,9 +48,17 @@ Adding a `type` label to an issue helps create visibility on the health of the p
 
 ### Support requests
 
-Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. In general we prefer that support requests are raised as [discussions](https://github.com/etcd-io/etcd/discussions), however if you identify that an issue is a support request please add the `type/support` or `type/question` label.
+Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. Support requests for etcd should be instead raised as [discussions](https://github.com/etcd-io/etcd/discussions).
 
-When responding to these requests please keep in mind the etcd [supported versions policy](https://etcd.io/docs/v3.6/op-guide/versioning/#service-versioning-and-supported-versions). If the request relates to an unsupported version and will require significant effort please instead kindly make the issue creator aware of the etcd support policy, encourage they update to a supported release, then close the issue, refer to [this example](https://github.com/etcd-io/etcd/issues/15796).
+If you identify that an issue is a support request please:
+
+ 1. add the `type/support` or `type/question` label.
+ 2. add a comment to inform the issue creator that discussions should be used instead and that this issue will be converted to a discussion.
```

Comment:
Please provide an ready template for the comment, example https://www.kubernetes.dev/docs/guide/issue-triage/#user-support-response-example

Reply:
Great suggestion, done 👍🏻